haiku-games/bebattle/bebattle-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Battle is a fun board strategy game"
+DESCRIPTION="BeBattle is a fun board strategy game."
+HOMEPAGE="https://github.com/HaikuArchives/BeBattle/"
+COPYRIGHT="1999-2000 Jonathan Villemure"
+LICENSE="MIT"
+REVISION="1"
+COMMIT="e44b8fd01f895e1ef068ab330abd3e492d95d369"
+SOURCE_URI="https://github.com/HaikuArchives/BeBattle/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="30be929f3fb9725fc8ca6f817fb75abd45c526436154d5dc397f39940b48f583"
+SOURCE_DIR="BeBattle-$COMMIT"
+
+ARCHITECTURES="x86_gcc2 !x86"
+
+PROVIDES="
+	bebattle = $portVersion
+	app:BeBattle = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:g++
+	makefile_engine
+	cmd:make
+	"
+
+BUILD()
+{
+	cd src
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cd src
+	cp objects.x86-cc2-release/BeBattle $appsDir
+	cp -r bmp $appsDir
```

Comment:
I would prefer to put these into a folder "BeBattle" under *$appsDir:
> cp objects.x86-gcc2-release/BeBattle $appsDir/BeBattle/
Maybe someone else can shed a light?

Reply:
I agree.
So it's:   `mkdir -p $appsDir/BeBattle`
etc.